lib/src/main/java/me/onebone/toolbar/AppBarContainer.kt

```diff
@@ -76,15 +67,15 @@ fun AppBarContainer(
 	modifier: Modifier = Modifier,
 	scrollStrategy: ScrollStrategy,
 	/** The state of a connected collapsing toolbar */
-	collapsingToolbarState: CollapsingToolbarState,
+	collapsingToolbarScaffoldState: CollapsingToolbarScaffoldState,
 	content: @Composable AppbarContainerScope.() -> Unit
 ) {
```

Comment:
Please revert contract(i.e. parameter or behavior) changes in `AppBarContainer.kt` as it is in maintenance mode and thus should not be changed until it is removed!